pkg/services/rpcsrv/subscription_test.go

```diff
@@ -100,8 +100,8 @@ func initCleanServerAndWSClient(t *testing.T, startNetworkServer ...bool) (*core
 			}
 		}
 		close(readerStopCh)
-		<-readerToExitCh
 		ws.Close()
+		<-readerToExitCh
```

Comment:
The only thing I'm worried about is the fact that we have a similar usage pattern in our WS client. It's not *exactly* similar, because it's `wsWriter` goroutine who is responsible for connection closing, but still, I don't quite understand the reason of WS read process hanging given the fact that we have timeout set. But we have tests for WS client closing process, and it passes on Windows, so likely there's no problem in real code.